src/main/java/com/mercadopago/client/merchantorder/MerchantOrderClient.java

```diff
@@ -50,6 +50,7 @@ public MerchantOrderClient(MPHttpClient httpClient) {
    * @param id merchant order id
    * @return merchant order
    * @throws MPException exception
```

Comment:
Para manter um padrão, o que acha de deixar todas as mensagens de exceção como "...an error if the request fails"?

Reply:
Feito